policyengine_us/variables/household/income/person/capital_gains/long_term_capital_gains_like_collectibles.py

```diff
@@ -0,0 +1,14 @@
+from policyengine_us.model_api import *
+
+
+class long_term_capital_gains_like_collectibles(Variable):
```

Comment:
Could we name this `long_term_small_business_stock_gains`?

If that's too imprecise, how about `long_term_capital_gains_non_collectibles` or `long_term_capital_gains_not_on_collectibles`? Or is `like` used in this way elsewhere?